compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -5999,6 +6232,12 @@ TR::Node *treetopSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier *
          }
       node->setFirst(child);
       }
+   if (!node->getFirstChild()->getOpCode().isNullCheck() && node->getFirstChild()->getOpCodeValue() == TR::PassThrough)
```

Comment:
Should this be `!node->getOpCode().isNullCheck()` ?

Reply:
I think so. @andrewcraik could you confirm?

Reply:
yup should be what @cathyzhyi suggested.

Reply:
Fixed in 805fc5b